independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Maven release ${{steps.metadata.outputs.current-version}}
         run: |
           git checkout -b release
-          mvn -B release:prepare -Prelease -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}}
+          mvn -B release:prepare -Prelease -DpreparationGoals="clean install" -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}}
```

Comment:
Can we add this to the project's pom.xml?

Reply:
That might be possible but is it worth the effort? Do we release in some other place?

Reply:
Actually here: https://github.com/quarkiverse/quarkiverse-parent/blob/main/pom.xml#L105, as in https://github.com/forge/core/blob/master/pom.xml#L244

Reply:
https://github.com/quarkiverse/quarkiverse-parent/pull/56

Reply:
Ok let's wait for the Quarkiverse parent release and I'll update this PR (parent version + build)  then 👍

Reply:
@ia3andy could you rebase this PR with the latest parent? Thanks!

Reply:
FWIW JReleaser 1.2.0 comes out by the end of August. We've adopted a 2 month release cadence for stable releases to help with planning https://github.com/jreleaser/jreleaser/discussions/873

Reply:
I've rebased, but I am not sure if we should merge this or not... Maybe it's better to have this until we have the actual fix ready?